src/annotator/integrations/test/pdf-metadata-test.js

```diff
@@ -94,6 +94,7 @@ class FakePDFViewerApplication {
     this.dispatchDOMEvents = domEvents;
     this.initialized = false;
     this.newFingerprintAPI = newFingerprintAPI;
+    this.downloadComplete = false;
```

Comment:
This was previously not initialized, relying on the "falsy" behavior of `undefined`.

Now we need to initialize to ensure the code branches that need to run are preserved.